charts/pega/charts/installer/templates/pega-installer-role.yaml

```diff
@@ -5,7 +5,13 @@ metadata:
   name: {{ template "installerJobReaderRole" }}
   namespace: {{ .Release.Namespace }}
 rules:
-- apiGroups: ["", "batch", "extensions", "apps"]
-  resources: ["jobs", "deployments", "statefulsets"]
+- apiGroups: ["apps"]
+  resources: ["deployments", "statefulsets"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["batch"]
+  resources: ["jobs"]
+  verbs: ["get", "watch", "list"]
+- apiGroups: ["extensions"]
+  resources: ["deployments"]
```

Comment:
@iMax-pp Is there any specific reason we are splitting the api groups and verbs?

Reply:
I splitted api groups and resources to allow for different verbs on each couple of api group / resource.